test/src/test/resources/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
 # Test docker image
-FROM ghcr.io/boozallen/aissemble-quarkus:1.8.0-SNAPSHOT
+FROM ghcr.io/boozallen/aissemble-quarkus:1.8.0
```

Comment:
S: Needs to be updated to 1.9.0-SNAPSHOT. Same with `test-app/apps/hello-world/Chart.yaml`

Reply:
ah.. good catch!